e2e/pages/adf/cardViewComponentPage.ts

```diff
@@ -174,4 +176,46 @@ export class CardViewComponentPage {
         }
     }
 
+    async getDateValue(): Promise<string> {
+        const dateValue = element(by.css('span[data-automation-id="card-date-value-date"]'));
+        return dateValue.getText();
+    }
+
+    async getDateTimeValue(): Promise<string> {
+        const dateTimeValue = element(by.css('span[data-automation-id="card-datetime-value-datetime"]'));
+        return dateTimeValue.getText();
+    }
+
+    async clearDateField(): Promise<void> {
+        const clearDateButton = element(by.css('mat-icon[data-automation-id="datepicker-date-clear-date"]'));
+        return BrowserActions.click(clearDateButton);
+    }
+
+    async clearDateTimeField(): Promise<void> {
+        const clearDateButton = element(by.css('mat-icon[data-automation-id="datepicker-date-clear-datetime"]'));
+        return BrowserActions.click(clearDateButton);
```

Comment:
no need to return the click promise. 
you can do ```await BrowserActions.click(clearDateButton);``` here so that the promise resolves here